### PR TITLE
Revamp Absurd

### PR DESCRIPTION
**Pull request by RobLoach**
- Brings in Boilerplate
- Brings in `test-jstransformer`
- Cleans up the processing method

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,48 +1,63 @@
...
-    throw new TypeError('jstransformer-absurd expects object or function');
+  // Check if we are to morph the object.
+  if (options.morph) {
+    absurd.morph(options.morph);
   }
-  if (isFunction) {
-    api = api(absurd, options);
+
+  // Process the input for the object.
+  if (typeof input == 'string' || input instanceof String) {
```
Comment (TimothyGu) [inline index.js]: Use duck typing when possible.

Also, isn't `input` not allowed to be anything but a string?

Comment (RobLoach) [inline index.js]: > Also, isn't input not allowed to be anything but a string?

Really not sure... Absurd has a lot of different inputs, and outputs.

**Comment by tunnckoCore** [comment]
:-1:

**Comment by RobLoach** [comment]
> :-1: 

If you state where your disagreement lies, then we could fix things to satisfy your needs.

**Comment by tunnckoCore** [comment]
https://github.com/jstransformers/jstransformer-gray-matter/pull/2#issuecomment-107745064
- will break my workflow, automation, sublime linter and fast fingers, :laughing:

**Comment by RobLoach** [comment]
> will break my workflow, automation, sublime linter and fast fingers, 

Move additions to [Boilerplate](https://github.com/jstransformers/boilerplate), and they will be brought downstream to all the transformers.

**Pull request closed without merging**